docs/Containerization.md

```diff
@@ -4,13 +4,14 @@ Docker is used to containerize our various micro-services
 which serves our combined application.
 
 We have various micro-services including:
-* MySQL Database
+* Redis Service
```

Comment:
I don't think Redis itself is considered a micro-service, while it is true that it is a service that we use.